### ISSUE TITLE
2019-11-20 tc016 M3SS comp engine error when parameterset set to "PARAMETERDISPLAYLIST"

### ISSUE THREAD

**Issue by jhhughes256** [tensfeldt/openNCA#44]
### Issue
When using tc016 and using the value of `"PARAMETERDISPLAYLIST"` for `parameterset` the computation engine crashes. This does not cause an error for tc010, tc011, tc013 or tc014. Errors for tc012 and tc015 are documented separately due to their unique error messages.

### Error

```
parameterset <- "PARAMETERDISPLAYLIST"
```
```
Error in value[[3L]](cond) : 
  Error in cltau(auctau = auctau[[d]], dose = dose_c[d]): object 'dose_c' not found
For SDEID 12596809 
7. stop(paste0(e, "For SDEID ", unique(data_data[, map_data$SDEID])[i])) at run_M3_SS_computation.R#2180
6. value[[3L]](cond) 
5. tryCatchOne(expr, names, parentenv, handlers[[1L]]) 
4. tryCatchList(expr, classes, parentenv, handlers) 
3. tryCatch({
    if (comp_required[["CMAXi"]]) {
        c_maxi <- list()
    } ... at run_M3_SS_computation.R#923
2. run_M3_SS_computation(data = merged_data, map = map_data, method = method, 
    model_regex = model_regex, parameter_list = parameter_list, 
    return_list = return_list) at run_computation.R#2247
1. run_computation(data = d, map = mct, flag = flags, parameterset = parameterset) 
```

**Comment by rudraya** [comment]
This issue was addressed in 02b2470, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
### Update

As of 02b2470 this issue still occurs, but with a slightly different error message. It is now having an error caused by `dose_c_i` rather than `dose_c`.

```
Error in value[[3L]](cond) : 
  Error in cltau(auctau = auctau[[d]], dose = dose_c_i[[d]]): object 'dose_c_i' not found
For SDEID 12596809 
7. stop(paste0(e, "For SDEID ", unique(data_data[, map_data$SDEID])[i])) at run_M3_SS_computation.R#2336
6. value[[3L]](cond) 
5. tryCatchOne(expr, names, parentenv, handlers[[1L]]) 
4. tryCatchList(expr, classes, parentenv, handlers) 
3. tryCatch({
    dof <- list()
    if (comp_required[["DOSECi"]] || comp_required[["DOSEC"]]) {
        dose_c_i <- list() ... at run_M3_SS_computation.R#972
2. run_M3_SS_computation(data = merged_data, map = map_data, method = method, 
    model_regex = model_regex, parameter_list = parameter_list, 
    return_list = return_list) at run_computation.R#2247
1. run_computation(data = d, map = mct, flag = flags, parameterset = parameterset)
```

**Comment by rudraya** [comment]
This issue was addressed in 96f6417, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
#44 appears to be resolved with commit 96f6417

Note that partial AUCs are still computed incorrectly for the special cases of interpolated concentrations utilized in partial AUCs. This will be reported separate.
